v2/bigtable-common/src/main/java/com/google/cloud/teleport/v2/bigtable/options/BigtableCommonOptions.java

```diff
@@ -226,7 +225,7 @@ interface ReadChangeStreamOptions extends BigtableCommonOptions.ReadOptions {
         description =
             "Bigtable change streams charset name when reading values and column qualifiers",
         helpText =
-            "The Bigtable change streams charset name when reading rowkeys, values, and column qualifiers. This option is used when message encoding is JSON.")
+            "The Bigtable change streams charset name when reading values and column qualifiers.")
```

Comment:
Looks like this description is some missing text. Should I use my description that has rowkeys and second line as well ?

Reply:
I'm not sure what you're asking here, but I would recommend shrinking the description to `Bigtable change streams charset name` and leaving the helptext